packages/core/src/common/keybinding.ts

```diff
@@ -153,44 +150,23 @@ export class KeybindingRegistry {
         return false;
     }
 
-}
-
-export class KeyEventEmitter implements Disposable {
-
-    private listener: EventListenerOrEventListenerObject;
-
-    constructor(
-        private commandRegistry: CommandRegistry,
-        private keybindingRegistry: KeybindingRegistry) {
-
-        this.listener = (event: any) => this.handleEvent(event);
-        window.addEventListener('keydown', this.listener, false);
-    }
-
-    dispose() {
-        window.removeEventListener('keydown', this.listener);
-    }
-
-    private handleEvent(event: KeyboardEvent): void {
-        if (!event.defaultPrevented) {
-            this.handleKey(KeyCode.createKeyCode(event), event);
+    run(event: KeyboardEvent): void {
```

Comment:
`handleEvent` sounds too generic for me, I like when the method name communities what it does, in this case, it runs the command matching the given event, I can rename `runCommand` or add the docstring.

Reply:
Haa ok I missed the idea that it would run the command.
I guess it may be too verbose but from when I wrote code on osx they had convenctions like:

runCommandFor(event: Keyboard event)

But I'll leave it to you doc string is fine